docs/source/traits_user_manual/notification.rst

```diff
@@ -491,6 +491,10 @@ with |@observe| is discouraged as this special trait may be removed when
 Identity comparison mode for container traits
 `````````````````````````````````````````````
 
+.. note:: This section is for Traits 6.1 only. Since Traits 6.2, the
```

Comment:
I'd suggest removing the relevant documentation instead. This version of the docs (in master) isn't going to see the light of day before we release Traits 6.2, and when we do release Traits 6.2 the documentation for 6.1 should still be available. So these docs are specific for 6.2

Reply:
Right, forgot we now have all the versions available online!

Reply:
> Right, forgot we now have all the versions available online!

We do still need to figure out how to make all those documentation versions easy to find for the user, though. (But that's a separate issue.)